README.md

```diff
@@ -4,7 +4,7 @@
 
 Multitenant AWS Dynamo supports the [AWS Dynamo Java API](http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/index.html?com/amazonaws/services/dynamodbv2/document/package-summary.html).
   
-You can write your application code against the Amazon DynamoDB interface as you would for any other application.  The implementation will manage storage of data by tenant.  You can tell the implementation how you want the data stored when you create the AmazonDynamoDB using the provided builders.  This library provides 3 multitenant builders that allow you to create your AmazonDynamoDB client.  Using these builders, you tell the implementation how you want to manage storage of tenant data.  The 3 basic implementations are `MTAmazonDynamoDBByAccount`, `MTAmazonDynamoDBByTable`, and `SharedTable`.  These implementations allow you to separate your tenants' data by AWS account, to separate your tenants' data by table name, or to store tenant data colocated in a set of predefined physical tables, respectively.  Builders may also be chained, allowing you to combine storage schemes.  In order for the implementation to manage multitenancy on your behalf, you provide it with an `MTAmazonDynamoDBContextProvider`.  The implementation calls back to the context provider whenever it needs to retrieve or store data.  Your implementation returns a string representing a tenant identifier of your choosing.
+You can write your application code against the Amazon DynamoDB interface as you would for any other application.  The implementation will manage storage of data by tenant.  You can tell the implementation how you want the data stored when you create the AmazonDynamoDB using the provided builders.  This library provides 3 multitenant builders that allow you to create your AmazonDynamoDB client.  Using these builders, you tell the implementation how you want to manage storage of tenant data.  The 3 basic implementations are `MtAmazonDynamoDBByAccount`, `MtAmazonDynamoDBByTable`, and `SharedTable`.  These implementations allow you to separate your tenants' data by AWS account, to separate your tenants' data by table name, or to store tenant data colocated in a set of predefined physical tables, respectively.  Builders may also be chained, allowing you to combine storage schemes.  In order for the implementation to manage multitenancy on your behalf, you provide it with an `MtAmazonDynamoDBContextProvider`.  The implementation calls back to the context provider whenever it needs to retrieve or store data.  Your implementation returns a string representing a tenant identifier of your choosing.
```

Comment:
DB -> Db in each of:
MtAmazonDynamoDBByAccount
MtAmazonDynamoDBByTable
MtAmazonDynamoDBContextProvider

Reply:
Fixed.  Thanks.